apps/meteor/app/api/server/v1/voip/rooms.ts

```diff
@@ -89,10 +89,11 @@ API.v1.addRoute(
 				token: String,
 				agentId: Match.Maybe(String),
 				rid: Match.Maybe(String),
+				direction: Match.Maybe(String),
```

Comment:
You can pass a custom validator here to do what you're doing on the if at 119

Reply:
Sure, its done :heavy_check_mark: